README.md

```diff
@@ -155,6 +157,75 @@ if __name__ == '__main__':
     IOLoop.current().run_sync(main)
 ```
 
+## `notebook-http` Mode
+
+The `KernelGatewayApp.api` command line argument can be set to `notebook-http`
+to allow a properly annotated notebook to be served via HTTP endpoints. A
+notebook cell must be annotated with a single line comment at the beginning
+describing the HTTP endpoint. An example python cell is:
+
+```python
+# GET /hello/world
+print("hello world")
+```
+
+This enables an HTTP GET on `/hello/world`. The [api_intro.ipynb](https://github.com/jupyter-incubator/kernel_gateway/blob/master/etc/api_examples/api_intro.ipynb)
+provides a full example notebook for reference.
+
+### Running
+The minimum number of arguments needed to run in HTTP mode are
+`--KernelGatewayApp.api=notebook-http` and
+`--KernelGatewayApp.seed_uri=some/notebook/file.ipynb`.
+
+You can run the kernel gateway in `notebook-http` mode from the Makefile:
+
+```
+make dev ARGS="--KernelGatewayApp.api='notebook-http' \
+--KernelGatewayApp.seed_uri=/srv/kernel_gateway/etc/api_examples/scotch_api.ipynb"
+```
+
+All of the notebooks in `etc/api_examples` are mounted into
```

Comment:
Clarify: with this make command. If they run the Docker container in general, the etc/api_examples don't show up automatically.